### ISSUE TITLE
False positive E1120 no-value-for-parameter for class method

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#40]
Originally reported by: **BitBucket: [mitar](http://bitbucket.org/mitar), GitHub: @mitar?**

---

I have the following code:

```
#!python
class DisplayComponent(components.FrontendComponent):
    @classmethod
    def get_urls(cls):
        return super(DisplayComponent, cls).get_urls()
```

And pylint complains that "No value passed for parameter 'cls' in function call". Which is of course wrong as this is done automatically.

pylint 0.28.0, 
astng 0.24.3, common 0.59.1

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/40
